js/gdm/authPrompt.js

```diff
@@ -87,7 +103,7 @@ var AuthPrompt = GObject.registerClass({
 
         this._initEntryRow();
 
-        let capsLockPlaceholder = new St.Label();
+        const capsLockPlaceholder = new St.Label();
```

Comment:
The opposite problem now: these `let` → `const` changes shouldn't be part of this commit :slightly_smiling_face: 

I'll point out which ones shouldn't have been changed below

Reply:
Good catch, I did a full search/replace and totally missed some occurrences were already upstream. Fixed now.